apps/okreads/browser/src/app/app.component.html

```diff
@@ -3,7 +3,7 @@
     <header class="header">
       <div class="header--content">
         <h1>okreads</h1>
-        <div>
+        <div tabindex="0" role="button">
```

Comment:
tabindex indicates that its elements can be focused, and where it participates in sequential keyboard navigation.